nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({extended:true}));
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/addition", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const addition = `${firstNumber +secondNumber }`
+  res.send(addition)
+});
+
+app.get("/calculator/subtraction", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const subtraction= `${firstNumber - secondNumber }`
+  res.send(subtraction)
+});
+
+app.get("/calculator/multiplication", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const multiplication= `${firstNumber * secondNumber }`
+  res.send(multiplication)
+});
+
+app.get("/calculator/division", (req, res) => { 
+  const firstNumber = parseInt(req.query.first);
+  const secondNumber = parseInt(req.query.second);
+  const division= `${firstNumber/secondNumber }`
+  res.send(division)
+});
+
+app.get("/calculator/add", (req, res) => { 
+const firstNumber = parseInt(req.query.first);
```

Comment:
It is good practice to make sure that your indentation is correct. Here, your code within the function is at the same indentation as the function definition. Instead, it should be like the functions above.